lib/models/github-login-model.js

```diff
@@ -54,6 +90,23 @@ export default class GithubLoginModel {
     this.didUpdate();
   }
 
+  async getScopes(host, token) {
+    if (atom.inSpecMode()) {
+      throw new Error('Attempt to check token scopes in specs');
+    }
+
+    const response = await fetch(host, {
+      method: 'HEAD',
```

Comment:
TIL -- I didn't realize `HEAD` is a http method.  (tho I guess there are all kindsa other weird ones like `PURGE`)